manim/gui/gui.py

```diff
@@ -0,0 +1,72 @@
+from pathlib import Path
+
+try:
+    import dearpygui.core
+    import dearpygui.demo
+    import dearpygui.simple
+
+    dearpygui_imported = True
+except ImportError:
+    dearpygui_imported = False
+
+
+from .. import config
+from ..utils.module_ops import scene_classes_from_file
+
+
+def configure_pygui(renderer, widgets, update=True):
+    if not dearpygui_imported:
+        raise RuntimeError("Attempted to use DearPyGUI when it isn't imported.")
```

Comment:
hmm, maybe something like `ImportError`?

Reply:
ImportError is reserved for situations where an error occurs while importing, but that's not what's happening in this situation.

Reply:
hmm, ok.